core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/sklearn/SklearnMLOp.scala

```diff
@@ -0,0 +1,73 @@
+package edu.uci.ics.texera.workflow.operators.sklearn
+
+import com.fasterxml.jackson.annotation.{JsonIgnore, JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort, PortIdentity}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+abstract class SklearnMLOp extends PythonOperatorDescriptor {
+  @JsonIgnore
+  var model = "from sklearn.dummy import DummyClassifier"
+
+  @JsonIgnore
+  var name = "Dummy Classifier"
+
+  @JsonProperty(value = "Target Attribute", required = true)
+  @JsonPropertyDescription("column in your dataset corresponding to target")
+  @AutofillAttributeName
+  var target: String = _
+
+  override def generatePythonCode(): String =
+    s"""$model
+       |from pytexera import *
+       |from sklearn.metrics import accuracy_score, f1_score, precision_score, recall_score
+       |class ProcessTableOperator(UDFTableOperator):
+       |    @overrides
+       |    def process_table(self, table: Table, port: int) -> Iterator[Optional[TableLike]]:
+       |        if port == 0:
+       |            self.model = ${model
+      .split(" ")
+      .last}().fit(table.drop("$target", axis=1), table["$target"])
+       |        else:
+       |            predictions = self.model.predict(table.drop("$target", axis=1))
+       |            auc = accuracy_score(table["$target"], predictions)
+       |            f1 = f1_score(table["$target"], predictions, average='micro')
+       |            precision = precision_score(table["$target"], predictions, average='micro')
+       |            recall = recall_score(table["$target"], predictions, average='micro')
+       |            print("Accuracy:", auc)
+       |            print("F1:", f1)
+       |            print("Precision:", precision)
+       |            print("Recall:", recall)
```

Comment:
I thought about this. can you print them in one line?

Reply:
Fixed.